localstack/services/s3/s3_listener.py

```diff
@@ -170,7 +170,7 @@ def get_event_message(event_name, bucket_name, file_name='testfile.txt', etag=''
 def send_notifications(method, bucket_name, object_path, version_id):
     bucket_name = normalize_bucket_name(bucket_name)
     for bucket, notifs in S3_NOTIFICATIONS.items():
-        if bucket == bucket_name:
+        if bucket.lower() == bucket_name.lower():
```

Comment:
Can we use this instead:
´´´
if normalize_bucket_name(bucket) == normalize_bucket_name(bucket_name):
´´´